### ISSUE TITLE
[0.9.0 Staging-1538] - Can only select one option for each of Voters and Vetoers

### ISSUE THREAD

**Issue by trickenso** [StrangeLoopGames/EcoIssues#16192]
It would be more flexible to be able to add an arbitrary list of titles, demographics and users to these lists.

For example, I might want to allow the World Leader, three specific users, and all admins to veto a particular kind of vote.

A single option for each of these would either harshly limit the possibilities, or require you to make specific titles/demographics specifically to accommodate the vote system.

**Comment by SlayksWood** [comment]
For example. You can create Title (in registrar) "Vetoers":
![image](https://user-images.githubusercontent.com/45708377/81378027-3cdee900-910f-11ea-9e15-a37f68e87ced.png)
and add anybody you want:
![image](https://user-images.githubusercontent.com/45708377/81378231-99da9f00-910f-11ea-9f1c-8f8c46772a11.png)

@johnkslg Is it intended bahavior or we can change it to list?

**Comment by trickenso** [comment]
The trouble I foresee is if you want different lists of voters/vetoers for different vote systems you would need a title for each, which might not be used for any other purpose. If you wanted to make a vote that allowed 'World Leader' and 'Council of Elders' to vote, or 'Senate' and 'Admins' to veto you would need a title for each combination.
Also, the number of titles you have is limited by your government offices. It's possibly a lot of government offices for titles that might only be used once.

**Comment by SlayksWood** [comment]
Registrar don't have any limits by slots.
To make it possible to select a list in the board of election is good. But it can be delayed

**Comment by trickenso** [comment]
Absolutely. Sorry for misunderstanding about slots - the interface for the registrar looks like it will only support 3 titles.

**Comment by WeaselDog** [comment]
This is a very interesting idea. I am going to move it to suggestions for later consideration.
